tests/Responses/Expected/shipment/shipment_index_response.json

```diff
@@ -0,0 +1,191 @@
+        "last": "/api/lagersystem/shipments?locale=en_US&page=1&limit=10",
+        "next": "/api/lagersystem/shipments?locale=en_US&page=1&limit=10"
+    },
+    "items": [
+        {
+            "id": @integer@,
+            "state": "ready",
+            "method": {
+                "code": "UPS",
+                "name": "UPS",
+                "description": "@string@"
+            },
+            "order": {
+                "id": @integer@,
+                "number": "000000001",
+                "channel": "US_WEB",
+                "currencyCode": "USD",
+                "localeCode": "en_US",
+                "state": "new",
+                "checkoutState": "completed",
+                "checkoutCompletedAt": "@string@.isDateTime()",
+                "paymentState": "authorized",
+                "payments": [
+                    {
+                        "state": "authorized",
+                        "method": {
+                            "code": "cash_on_delivery",
+                            "name": "Cash on delivery",
+                            "description": "@string@"
+                        }
+                    }
+                ],
+                "shippingAddress": {
+                    "firstName": "Igor",
+                    "lastName": "Mukhin",
+                    "countryCode": "UA",
+                    "street": "Main street",
+                    "city": "Odesa",
+                    "postcode": "65000",
+                    "phoneNumber": "380951234567"
+                },
+                "billingAddress": {
+                    "firstName": "Igor",
+                    "lastName": "Mukhin",
+                    "countryCode": "UA",
+                    "street": "Main street",
+                    "city": "Odesa",
+                    "postcode": "65000",
+                    "phoneNumber": "380951234567"
+                },
+                "adjustments": {
+                    "UPS": {
+                        "amount": 500,
+                        "neutral": false
+                    },
+                    "TAX_EU": {
+                        "amount": 399,
+                        "neutral": true
+                    },
+                    "PROMO_UNIT_X": {
+                        "amount": -198,
+                        "neutral": false
+                    }
+                },
+                "itemsTotal": 3800,
+                "total": 4300,
+                "adjustmentsTotal": 302,
+                "customer": {
+                    "id": @integer@,
+                    "firstName": "Igor",
+                    "lastName": "Mukhin",
+                    "email": "igor.mukhin@gmail.com",
+                    "birthday": "@string@.isDateTime()",
+                    "gender": "m",
+                    "phoneNumber": "380951234567"
+                }
+            },
+            "units": [
+                {
+                    "shippable": {
```

Comment:
What's the reason for this `shippable` array?

Reply:
This is actually product variant. But from shipments POV it called `shippable`
We need it (or at least link to product variant endpoint) just to know what exactly unit we have.